spec/support/capybara_webkit.rb

```diff
@@ -1,5 +0,0 @@
-Capybara.javascript_driver = :webkit
```

Comment:
Should `capybara-webkit` be removed from the Gemfile, too?

Reply:
Yes, this was done previously but good thinking 😉